### PR TITLE
Kill world_vision custom code

### PR DESCRIPTION
**Pull request by orangejenny**
Based on google analytics, the world_vision custom reports (mother_report and child_report) haven't been accessed since September 2016, which fits with Ethan's observation that wvindia2 has been on community since April 2017 and hasn't submitted a real form since then.

@nickpell / @esoergel

### CODE REVIEW

**Review by nickpell — APPROVED** [review]

File: corehq/apps/reports/views.py [hunk 1]
```diff
@@ -237,9 +236,6 @@ def can_view_attachments(request):
 @login_and_domain_required
 @location_safe
 def default(request, domain):
-    if domain in WORLD_VISION_DOMAINS and settings.DOMAIN_MODULE_MAP.get(domain):
-        from custom.world_vision.reports.mixed_report import MixedTTCReport
-        return HttpResponseRedirect(MixedTTCReport.get_url(domain))
     return HttpResponseRedirect(reverse(MySavedReportsView.urlname, args=[domain]))
```
Comment (esoergel) [inline corehq/apps/reports/views.py]: Can this whole view be removed now?

**Review by esoergel — COMMENTED** [review]
Awesome.  There are a bunch of fluff Documents and SqlData classes in here - I don't remember how to deal with those exactly, but I think something needs to be done to get rid of those.

File: corehq/apps/reports/views.py (hunk 1 above)
Comment (orangejenny) [reply corehq/apps/reports/views.py]: yep - updated

**Comment by orangejenny** [comment]
@esoergel Emord got rid of the pillows in https://github.com/dimagi/commcare-cloud/pull/1901 so I think the fluff documents are okay. Any idea who would have more knowledge of what to do about the sqldata part?

**Comment by esoergel** [comment]
@orangejenny maybe Simon?

**Comment by emord** [comment]
> Emord got rid of the pillows in dimagi/commcare-cloud#1901 so I think the fluff documents are okay. 

@orangejenny Those aren't really related. removing pillows just stops it from listening to kafka/writing to couch. all old docs would still be there.

From what I can tell these `fluff.IndicatorDocument`  write directly to SQL, but don't have associated migrations because they use sqlalchemy/alembic to sync their definitions. I think you'll want to use https://github.com/dimagi/commcare-hq/blob/master/corehq/ex-submodules/fluff/management/commands/wipe_fluff_table.py or write a migration to drop the tables

**Comment by orangejenny** [comment]
> There are a bunch of fluff Documents and SqlData classes in here - I don't remember how to deal with those exactly, but I think something needs to be done to get rid of those.

@snopoke do you know what I need to do to get rid of the sqldata classes?

**Review by snopoke — APPROVED** [review]
What do you think about deleting the fluff tables as well?

**Comment by orangejenny** [comment]
@snopoke Yeah, I'll wipe the fluff tables in a shell after merging this.

Will merge Monday, Friday nights have been bad luck lately.

**Comment by orangejenny** [comment]
Ran `wipe_fluff_tables` on prod for `WorldVisionMotherFluffPillow`, `WorldVisionChildFluffPillow`, and `WorldVisionHierarchyFluffPillow`

**Pull request merged**